src/Components/ADT3DSceneBuilder/Internal/Behaviors/Widgets/WidgetBuilders/DataHistoryWidgetBuilder/Internal/TimeSeriesFormCallout.tsx

```diff
@@ -169,6 +163,12 @@ const TimeSeriesFormCallout: React.FC<IProp> = ({
                     value={seriesToEdit?.label}
                     onChange={handleLabelChange}
                 />
+                <TextField
+                    placeholder={'e.g. m, m², kg, s, °C'}
```

Comment:
idk if e.g. is kosher to have since that doesn't translate?

Reply:
not sure if the translation will/should translate to the units but moved it to translation

Reply:
okay the translations looks good, not sure about Chinese though